Content.Server/GameObjects/Components/Mobs/Body/Prototypes/BodyPrototype.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using YamlDotNet.RepresentationModel;
+using Robust.Shared.Serialization;
+using Robust.Shared.Prototypes;
+using Robust.Shared.IoC;
+using Robust.Shared.Utility;
+using Robust.Shared.Interfaces.Reflection;
+
+namespace Content.Server.GameObjects.Components.Mobs.Body
+{
+    /// <summary>
+    ///    Core of the mobcode. It glues all the shitcode with limbs, organs 
+    ///    and body functions together with DAMAGE, making frankensteins that we call Mobs
+    /// </summary>
+
+    [Prototype("bodyTemplate")]
+    public class BodyPrototype : IPrototype, IIndexedPrototype
+    {
+#pragma warning disable CS0649
+        [Dependency]
+        protected IReflectionManager reflectionManager;
+#pragma warning restore
+
+        string IIndexedPrototype.ID => Id;
+
+        public string Name { get; private set; }
+        public string Id { get; private set; }
+        string ClassName;
+
+        public List<string> LimbPrototypes { get; private set; }
+
+        void IPrototype.LoadFrom(YamlMappingNode mapping)
+        {
+            var obj = YamlObjectSerializer.NewReader(mapping);
+            Id = obj.ReadDataField("id", "");
+            Name = obj.ReadDataField("name", "");
+            obj.DataField(ref ClassName, "className", "");
+            LimbPrototypes = new List<string>();
+            foreach (var limbMap in mapping.GetNode<YamlSequenceNode>("limbs").Cast<YamlMappingNode>())
```

Comment:
Do you have any reason the data isn't laid out like this:

```yml
limbs:
- A
- B
- C
```

Having to specify `map:` here makes loading more annoying and makes it easier to write. If you simplify it you can do `ReadDataField<List<string>>(...)`